client/src/components/navLink/NavLink.jsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import classes from './navLink.module.css';
 
-function NavLink() {
-  return <div></div>;
+function NavLink({ text, path, theme }) {
+  return (
+    <Link className={classes.link} data-theme={theme} to={path}>
+      {text}
```

Comment:
It's rather a not flexible design. Later you may find yourself adding more props coz your NavLink starts to grow. You can simply start with children.